File: backend/src/main/java/com/backend/petplace/global/config/ImportProperties.java
package com.backend.petplace.global.config;

import org.springframework.boot.context.properties.ConfigurationProperties;

@ConfigurationProperties(prefix = "import")
public record ImportProperties(
    boolean enabled,
    String baseUrl,
    String serviceKey,
    int pageSize,
    long sleepMs,
    int maxInMemoryMb
) {}

File: backend/src/main/java/com/backend/petplace/global/config/ImportProperties.kt
package com.backend.petplace.global.config

import org.springframework.boot.context.properties.ConfigurationProperties

@ConfigurationProperties(prefix = "import")
data class ImportProperties(
  val enabled: Boolean,
  val baseUrl: String,
  val serviceKey: String,
  val pageSize: Int,
  val sleepMs: Long,
  val maxInMemoryMb: Int
)
